src/components/admin/form-components/SearchBar.jsx

```diff
@@ -97,23 +127,25 @@ export class SearchBar extends React.Component {
 
 SearchBar.propTypes = {
   handleClick: PropTypes.func.isRequired,
-  suggestions: PropTypes.arrayOf(
-    PropTypes.shape({
-      title: PropTypes.string.isRequired,
-      slug: PropTypes.string,
-      id: PropTypes.number.isRequired,
-    }),
-  ),
-  updateSuggestions: PropTypes.func.isRequired,
+  getSuggestions: PropTypes.func.isRequired,
+  mode: PropTypes.string.isRequired,
   fullWidth: PropTypes.bool,
   disabled: PropTypes.bool,
   enableAdd: PropTypes.bool,
-  category: PropTypes.string.isRequired,
+  slugify(props, propName) {
+    if (
+      props.enableAdd === true &&
+      (props[propName] === undefined || typeof props[propName] !== 'function')
+    ) {
+      return new Error('Please provide a slugify function!');
```

Comment:
You can do as the example in the prop types docs and provide the component and prop name:
![screenshot from 2018-04-09 17-33-43](https://user-images.githubusercontent.com/18149242/38501004-f9aa869c-3c1c-11e8-8183-419b85d4d030.png)